warehouse/schema.go

```diff
@@ -8,7 +8,7 @@ import (
 	"time"
 )
 
-type exportSchema struct {
+type bundleSchema struct {
```

Comment:
Nit: can you add a comment (here or in the README) that explains the difference between the BundleSchema and the ExportTableSchema? It's not intuitive (for someone who wasn't part of our conversation yesterday!), and it's unclear at first glance how `WarehouseField` relates to them. Thanks!